### ISSUE TITLE
Ensure full compatibility of the lobster framework with octave

### ISSUE THREAD

**Issue by jdbancal** [replab/replab#160]
This requires small adjustments

**Comment by denisrosset** [comment]
@jdbancal I've seen that you made small commits; do things work well now?

By the way, why were the parent constructor calls not working in Octave? They seemed to work in other classes (I'm def. getting grey hairs over Octave/Matlab compat)

**Comment by jdbancal** [comment]
Yes, things seem to work very well, only small changes were necessary.

The problem was not with calling the parent constructor: the parent constructor gets called without problem. The thing was that the parent constructor was calling an abstract method that was overloaded (i.e. implemented) in the child class. Actually, also this part seemed to work fine, the problem showed up when the method in the child class was trying to initialize an attribute of the child class: at this point Octave complained that a class method, which is not the constructor, was trying to initialize a property of the object (although it was fundamentally called from the class constructor itself, just indirectly).

My impression is that Octave probably just lost track of the reason it got to this point and put an error by default. This confirms other examples I experienced where class construction seems not to be very robust in Octave (e.g. in some cases it is possible to instantiate an object without ever going through the constructor (!!)). Anyway, for lobster, we didn't really need this intricate construction: we have just 4-5 classes which inherit `replab.lobster.Node`, and they can be easily initialized from their own constructors without too much fuss, so I just modified that.
